### ISSUE TITLE
note() eats words in LaTeX quotes

### ISSUE THREAD

**Issue by NilsEnevoldsen** [derekwolfson/tex3pt#13]
```
note("\`\`foo'' bar")
```

yields

```
\Figtext{{\footnotesize bar}} %%TABLE NOTE
```

The backticks are escaped for Stata's sake, not LaTeX's.

**Comment by NilsEnevoldsen** [comment]
Titles fixed by 4d59d0c. I forgot to check notes, silly me.

**Comment by NilsEnevoldsen** [comment]
Fixed by 43b59cd. Kind of ugly, with lots of `macval()` statements. Is there a better way?
